src/custom-elements/src/components/date/date.tsx

```diff
@@ -190,6 +203,17 @@ export class Date {
   }
 
   componentWillLoad(): void {
+    if (this.excludedDates) {
```

Comment:
Please, change to a more robust code so it will handle change of attributes once the component is already loaded:
```JavaScript
@Watch('excludedDates')
updateExcludedDates() {
	if (this.excludedDates) {
		this.excludedDates.split(',').map(date => {
			this.excludedDatesArray.push(date);
		});
	} else {
		this.excludedDatesArray = [];
	}
}

@Watch('excludedWeedDays')
updateExcludedWeekDays() {
	if (this.excludedWeedDays) {
		this.excludedWeedDays.split(',').map(date => {
			this.excludedWeedDaysArray.push(date);
		});
	} else {
		this.excludedWeedDaysArray = [];
	}
}

componentWillLoad(): void {
	this.updateExcludedDates();
	this.updateExcludedWeekDays();
}
```

Reply:
Thanks, I also added step of emptying the `excludedDatesArray` and `excludedWeekdaysArray` before pushing new days when attributes are updated